2-mandatory/1-create-functions.js

```diff
@@ -3,15 +3,19 @@ Write a function that:
 - Accepts an array as a parameter.
 - Returns a new array containing the first five elements of the passed array.
 */
-function first5() {
+function first5(arrayFirstFive) {
+  return arrayFirstFive.slice(0, 5);
 }
 
 /*
 Write a function that:
 - Accepts an array as a parameter.
 - Returns a new array containing the same elements, except sorted.
 */
-function sortArray() {
+function sortArray(array) {
+  const newArray = [...array];
```

Comment:
Good use of spread to avoid making a copy by reference. You could also use Array.from(array) which might be a bit easier to understand.